js/src/logger.ts

```diff
@@ -943,7 +945,7 @@ export type InitOptions = {
  * @param options.description An optional description of the experiment.
  * @param options.dataset (Optional) A dataset to associate with the experiment. You can pass in the name of the dataset (in the same project) or a
  * dataset object (from any project).
- * @param options.update If the experiment already exists, continue logging to it.
+ * @param options.open If the experiment already exists, open it instead of creating a new one.
```

Comment:
Hmm I'm not sure I love the idea of conflating `open` and `update`. I was hoping that this would serve as the implementation of the `GET /v1/experiment/{experiment_id}` REST endpoint, which just lets you read an experiment, without possibly mutating it. And `update` remains the implementation of `PUT /v1/experiment`.

Now if someone explicitly specifies a `BaseExperiment` which doesn't exist, it'll just create a random empty experiment, which seems like a weird unintended side-effect. And if the experiment does exist, won't we overwrite existing fields like `isPublic`?

Reply:
BTW if you are looking for an endpoint to retrieve an existing experiment by name, look no further than `GET /v1/experiment`, which lets you specify a query param `experiment_name`.

Reply:
Ok NVM, I hit a bug with my approach and am convinced :). Fixing

Reply:
Yeah I feel strongly about this one because I think it would make it more confusing and difficult to cleanly expose those two functions separately without making a backwards-incompatible change, or creating a function which sounds similar to `open` but does something different. I'm just thinking from the perspective of creating parity between the existing SDKs and the new REST API.